sw/device/tests/otbn/otbn_ecdsa_p256_test.c

```diff
@@ -241,19 +279,24 @@ static void test_ecdsa_p256_roundtrip(void) {
   check_data("signature_r", signature_r, kExpectedSignatureR, 32);
   check_data("signature_s", signature_s, kExpectedSignatureS, 32);
 
+  // Cleanup OTBN variables.
+  p256_ecdsa_clean(&otbn_ctx);
```

Comment:
To make the test slightly more robust, could you change this and the other places where you use `p256_ecdsa_clean()` to code which clears the data memory completely? `CHECK(otbn_zero_data_memory(&otbn_ctx) == kOtbnOk)` should do the trick.

Reply:
Great :) I have used this function instead. I noticed that I also need to reload the OTBN app again for verification.
Please have a look :)